### PR TITLE
Taylor- adding password toggle

### PR DESCRIPTION
**Pull request by TaylorDJones11**
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Other
  
  
## Description of what this Pull Request does
- Allows users to toggle between password visibility
- When the input value is erased, the icon is set back to default

## Related Tickets & Documents
-Closes: #98 

  
## QA Instructions, Screenshots, Recordings
Password Toggle

- I'm having issues with setting the input/svg back to the default if the user erases the input while the password is visible

### CODE REVIEW

File: src/composables/passwordToggleFunction.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    if (inputValue !== "") {
+      setIsPasswordVisible(!isPasswordVisible);
+      setIsDefaultPassword(false);
+    } else {
+      setIsPasswordVisible(false);
+      setIsDefaultPassword(true);
+    }
+  };
+
+  const eyeIcon = isPasswordVisible ? (
```
Comment (annonymousauthority) [inline src/composables/passwordToggleFunction.js:25]: Put this in a useEffect hook

And it should watch fo change in `isPassword` `[isPasswordVisible]`

File: src/composables/passwordToggleFunction.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+const PasswordToggle = () => {
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+  const [inputValue, setInputValue] = useState("");
+  const [isDefaultPassword, setIsDefaultPassword] = useState(true);
+
+  const handleInputChange = (e) => {
+    const value = e.target.value;
+    setInputValue(value);
+    setIsDefaultPassword(value === "" || !isPasswordVisible);
```
Comment (annonymousauthority) [inline src/composables/passwordToggleFunction.js:12]: Do a check also here `setIsPasswordVisible(false)` if value === ""

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Please take the passWordToggleFunction to the component folder

**Comment by TaylorDJones11** [comment]
> Please take the passWordToggleFunction to the component folder

Everything has been resolved, but the dark/light mode is a mess 🥲

**Review by annonymousauthority — APPROVED** [review]
Referral to colors

**Pull request merged**